packages/declaration/lua/language-server/global.def.lua

```diff
@@ -25,6 +25,9 @@ function broadcastToAll(message, color) end
 ---@return boolean
 function broadcastToColor(message, playerColor, messageColor) end
 
+---@return table
```

Comment:
Could directly return `tts__PlayerColor[]` to get the information about the colors directly.